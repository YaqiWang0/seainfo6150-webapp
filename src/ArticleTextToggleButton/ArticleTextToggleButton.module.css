.button {
    box-shadow: -1px -1px grey, 1px 1px black;
    border: 1px solid black;
    border-radius: 5px;
    color: white;
    background-color: #c7090b;
    padding: 7px;
    font-size: 12px;
    margin-top: 15px;
}