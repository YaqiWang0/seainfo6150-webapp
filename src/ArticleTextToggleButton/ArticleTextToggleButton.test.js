import React from "react";
import ArticleTextToggleButton from "./ArticleTextToggleButton";

describe("ArticleListItem tests", () => {
    it("renders correctly", () => {
        const data = "Show more";
        const {container} = render(<ArticleTextToggleButton buttonText = {data}/>);
        expect(container ).toMatchSnapshot();
    });
});