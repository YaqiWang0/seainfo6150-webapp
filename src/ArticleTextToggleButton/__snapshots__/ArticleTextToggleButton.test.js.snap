// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleListItem tests renders correctly 1`] = `
<div>
  <button
    class="button"
  >
    Show more
  </button>
</div>
`;
